constructor_io/modules/catalog.py

```diff
@@ -0,0 +1,164 @@
+'''Catalog Module'''
+
+from urllib.parse import quote, urlencode
+
+import requests as r
+
+from constructor_io.helpers.exception import ConstructorException
+from constructor_io.helpers.utils import (clean_params, create_auth_header,
+                                          create_request_headers,
+                                          throw_http_exception_from_response)
+
+
+def _create_query_params_and_file_data(parameters):
+    '''Create query params and file data'''
+
+    query_params = {}
+    file_data = {}
+
+    if parameters:
+        section = parameters.get('section')
+        notification_email = parameters.get('notification_email')
+        force = parameters.get('force')
+        items = parameters.get('items')
+        variations = parameters.get('variations')
+        item_groups = parameters.get('item_groups')
+
+        if section:
+            query_params['section'] = section
+
+        if notification_email:
+            query_params['notification_email'] = notification_email
+
+        if force:
+            query_params['force'] = force
+
+        if items:
+            file_data['items'] = ('items.csv', items)
+
+        if variations:
+            file_data['variations'] = ('variations.csv', variations)
+
+        if item_groups:
+            file_data['item_groups'] = ('item_groups.csv', item_groups)
+
+
```

Comment:
I don't think this extra line is the norm, but maybe

Reply:
It is not. Good catch